### DISCUSSION TITLE
Can WIKI be enabled on this git?

### DISCUSSION THREAD

**Discussion by shooking** [RomanKubiak/ctrlr#296] [Ideas]
And a few folks be given access to modify it?

On my own github I use the Wiki to distil information - and I notice there are some great folks who are able to write concise information. What I see are a series of great discussions here, and some make it to the Issues (plus old info from historical Wordpress).

Would be great to get the info added into a book-like reference and the github wiki seems as good a vehicle as any?

**Comment by RomanKubiak** [comment]
yes I will enable Wiki
